lms/static/js/learner_dashboard/views/program_header_view_2017.js

```diff
@@ -0,0 +1,76 @@
+(function(define) {
+    'use strict';
+
+    define(['backbone',
+        'jquery',
+        'edx-ui-toolkit/js/utils/html-utils',
+        'text!../../../templates/learner_dashboard/program_header_view_2017.underscore',
+        'picturefill',
+        'text!/static/images/programs/micromasters-program-details.svg',
```

Comment:
That's weird, I didn't think we have to do anything special because the same ```text!``` plugin is being used for underscore templates. Maybe the karma.conf isn't aware of the folder the images are in?